pages/thanks.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import Head from 'next/head';
+import Nav from '../components/Nav';
+import Footer from '../components/Footer';
+import Link from 'next/link';
+import Cookies from 'js-cookie';
+
+function Thanks() {
+  function resetCookies() {
+    Cookies.remove('cart');
+    Cookies.remove('total');
+  }
+  return (
+    <div>
+      <div className="thankPage">
+        <Head>
+          <title>Ecommerce-Store</title>
+          <link rel="icon" href="/favicon.png" />
+        </Head>
+        <Nav />
+        <hr />
+
+        <div className="thanks">
+          <h1>THANK YOU!</h1>
+          <h4>Your order was completed successfully</h4>
+          <Link href="/about">
+            <a>
+              <button onClick={resetCookies}>To home</button>
```

Comment:
Same thing here with `button` inside of a `Link`, probably don't want this. You could just have a button instead of the `Link`, and then inside the onClick function, you could use `Router.push`:

https://nextjs.org/docs/api-reference/next/router#routerpush